base/templates/base/includes/header.html

```diff
@@ -181,40 +203,54 @@
                         <li class="dropdown">
                             <a href="#" data-ga-category="main-nav-links" data-ga-action="pulldown" data-ga-label="Collections &amp; Exhibits" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Collections <span class="hidden-sm">&amp; Exhibits</span> <span class="caret"></span></a>
                             <ul class="dropdown-menu twocol-drop">
-                                <li>
-                                    <ul class="list-unstyled col-sm-6 ul-left">
-                                        <li class="twocol-head">Collections</li>
-                                        <li role="separator" class="divider visible-xs"></li>
-                                        <li><a href="/collex/" data-ga-category="main-nav-links" data-ga-action="click" data-ga-label="Collections &amp; Exhibits &gt; Notable Collections">Notable Collections</a></li>
-                                        <li><a href="/collex/?digital=on&view=collections" data-ga-category="main-nav-links" data-ga-action="click" data-ga-label="Collections &amp; Exhibits &gt; Digital Collections">Digital Collections</a></li>
-                                        <li><a href="/collex/?view=subjects" data-ga-category="main-nav-links" data-ga-action="click" data-ga-label="Collections &amp; Exhibits &gt; Collecting Areas by Subject">Collecting Areas by Subject</a></li>
-                                        <li><a href="/collex/collections/other-local-collections/" data-ga-category="main-nav-links" data-ga-action="click" data-ga-label="Collections &amp; Exhibits &gt; Other Local Collections">Other Local Collections</a></li>
-                                        <li role="separator" class="divider hidden-xs"></li>
-                                        <li class="twocol-head">Exhibits</li>
-                                        <li role="separator" class="divider visible-xs"></li>
-                                        <li><a href="/collex/?view=exhibits" data-ga-category="main-nav-links" data-ga-action="click" data-ga-label="Collections &amp; Exhibits &gt; All Exhibits">All Exhibits</a></li>
-                                        <li><a href="/collex/?digital=on&view=exhibits" data-ga-category="main-nav-links" data-ga-action="click" data-ga-label="Collections &amp; Exhibits &gt; Web Exhibits">Web Exhibits</a></li>
-                                    </ul>
 
-                                    <ul class="list-unstyled col-sm-6">
-                                        <li class="twocol-head">Research Centers</li>
-                                        <li role="separator" class="divider visible-xs"></li>
+                                <div class="col-sm-6 ul-left">
```

Comment:
I think a `div` as a child of a `ul` might not be allowed. That's my understanding or it. You might want to double check on that.

Reply:
Good catch. I did this for two different pulldowns and resolved this in two different ways depending on the content of the lists.